Utils/Extensions/Keyboard.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  Keyboard.swift
+//  AirlinesApplication
+//
+//  Created by HendEl-Mahdy on 24/08/2024.
+//
+
+import Foundation
+import UIKit
+
+extension UIViewController{
```

Comment:
- is think there is a retain cycle here (memory leaks)
- ```
deinit {
    NotificationCenter.default.removeObserver(self)
}
```

Reply:
Already done in AddAirlinesViewController line 78